lib/memo_wise.rb

```diff
@@ -81,4 +81,28 @@ def reset_all_memo_wise
     @_memo_wise_keys.clear
     @_memo_wise.clear
   end
+
+  # If no block is given, will set the value to nil
+  def preset_memo_wise(method_name, *args)
+    not_memoized_name = :"_not_memoized_#{method_name}"
+    unless self.class.private_method_defined?(not_memoized_name)
```

Comment:
Thoughts on this approach to checking if a method has been memoized? Another option I was considering is just storing a set of memoized methods and checking for presence of `method_name` in that set.